commit 57b01008442fdcea108fac3d1dbb8a56b969364a
Author: Foo Monkey <pedia@tara.(none)>
Date:   Mon Mar 23 11:17:43 2009 +0000

    Mark tables that are bigger than 240 with a red background color
    
    only works on browser implementing querySelectorAll... Firefox
    3.5 will do it, current webkit browsers already implement it

diff --git a/app/views/layout/_fixup.html.haml b/app/views/layout/_fixup.html.haml
index d11e34d..d53035d 100644
--- a/app/views/layout/_fixup.html.haml
+++ b/app/views/layout/_fixup.html.haml
@@ -19,4 +19,10 @@
            }
        }
 
-
+       // Mark too big content
+       var nodes = document.querySelectorAll("table")
+       for (x in nodes) {
+            if (nodes[x].clientWidth + nodes[x].clientLeft > 240) {
+                nodes[x].style.setProperty("background-color", "red");
+            }
+       }
diff --git a/public/stylesheets/application.css b/public/stylesheets/application.css
index e78ba3e..add4329 100644
--- a/public/stylesheets/application.css
+++ b/public/stylesheets/application.css
@@ -24,6 +24,7 @@ html {
   font-size: 67.5%; }
 
 body {
+  width: 240px;
   line-height: 1;
   font-size: 1.2em;
   color: #000;

